src/js/tasks-function.js

```diff
@@ -0,0 +1,223 @@
+      const moveBtn = [];
+      moveBtn[elem.index] = document.createElement('button');
+      moveBtn[elem.index].setAttribute('id', elem.index);
+      moveBtn[elem.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical ellipsis"></i>';
+
+      // const deleteBtn = [];
+      // deleteBtn[elem.index] = document.createElement('i');
+      // deleteBtn[elem.index].setAttribute('id', elem.index);
+      // deleteBtn[elem.index].classList.add('fa-regular fa-trash-can delete');
+      li[elem.index].append(
+        inputBox[elem.index],
+        label[elem.index],
+        // deleteBtn[elem.index],
+        moveBtn[elem.index],
+      );
+      taskList.append(li[elem.index]);
+      li[elem.index].addEventListener('click', () => {
+        li[elem.index].style.backgroundColor = '#ff98008f';
+
+        label[elem.index].addEventListener('click', (e) => {
+          e.target.nextSibling.innerHTML = '<i class="fa-solid fa-trash-can"></i>';
+          e.target.nextSibling.style.cursor = 'pointer';
+
+          // deleteBtn[elem.index].classList.add('active');
+          e.target.nextSibling.addEventListener('click', () => {
+            li[elem.index].remove();
+            this.removetask(elem.index);
+          });
+        });
+      });
+      li[elem.index].addEventListener('mouseleave', (e) => {
+        this.updateTask(e.target.id, e.target.innerText);
+      });
+      inputBox[elem.index].addEventListener('change', (e) => {
+        const status = new TaskStatus();
+        if (e.target.checked === true) {
+          status.checked(this.taskData[elem.index]);
+        } else {
+          status.unchecked(this.taskData[elem.index]);
+        }
+        this.updateTask(
+          e.target.nextSibling.id,
+          e.target.nextSibling.innerText,
+        );
+      });
+      if (this.taskData[elem.index].completed === true) {
+        inputBox[elem.index].setAttribute('checked', 'checked');
+        li[elem.index].classList.add('checked');
+        label[elem.index].style.textDecoration = 'line-through';
+      } else if (this.taskData[elem.index].completed === false) {
+        inputBox[elem.index].removeAttribute('checked');
+        label[elem.index].style.textDecoration = 'none';
+      }
+    });
+  };
+
+  addtask = (description, completed, index) => {
+    const newtask = new Tasks(description, completed, index);
+    this.taskData.push(newtask);
+    if (typeof window !== 'undefined') {
+      localStorage.setItem('TodoListDB', JSON.stringify(this.taskData));
+    }
+  }
+
+  removetask = (item) => {
+    const key = item;
+    if (this.taskData.length === 1) {
+      this.taskData = [];
+    } else {
+      this.taskData.splice(key, 1);
+    }
+    this.taskData.forEach((elem, index) => {
+      elem.index = index;
+    });
+    if (typeof window !== 'undefined') {
+      localStorage.setItem('TodoListDB', JSON.stringify(this.taskData));
+      this.displayTask();
+    }
+  }
```

Comment:
- [ ] Kindly add a way to remove a task. This is what I saw when I tried to click on a task and I found no way to delete the task.

![image](https://user-images.githubusercontent.com/62290545/214618562-9cdb174f-7b1e-4ee9-a270-10c2c6f502fc.png)